internal/app/app.go

```diff
@@ -108,7 +111,7 @@ func NewServiceRepository(ctx context.Context, logger log.Logger) model.ServiceR
 // NewFileStorage creates a new connection with the file storage for postmortem document
 func NewFileStorage(ctx context.Context, logger log.Logger) filestorage.Driver {
 	configuredFileStorage := config.Env.FileStorage
-	logger.Info(
+	logger.Debug(
```

Comment:
I would actually keep this one as Info, since it is just a one-line log, runs only once, in the beginning, and brings a relevant information (the configured file storage)